src/ansys/geometry/core/designer/body.py

```diff
@@ -819,6 +819,16 @@ def unite(self, other: Union["Body", Iterable["Body"]], keep_other: bool = False
         """
         return
 
+    def combine_merge(self, other: Union["Body", list["Body"]]) -> None:
+        """Combine this body with another body or bodies, merging them into a single body.
+
+        Parameters
+        ----------
+        other : Union[Body, list[Body]]
+            The body or list of bodies to combine with this body.
```

Comment:
Is the "other" body consumed? Do both bodies (self, and other) still exist after the operation? Do both bodies disappear and a new one (new ID et al.) gets created? I think we should clarify that in the docstring as a note.